test/handler/test_brakeman_handler.vader

```diff
@@ -7,20 +7,23 @@ After:
     call ale#linter#Reset()
 
 Execute(The brakeman handler should parse JSON correctly):
+  cd! /testplugin/test/ruby_fixtures/valid_rails_app/app/models
```

Comment:
I might add a second `cd` line so the test runs when you run the file directly with `:Vader`. This is good. :+1: